### PR TITLE
instrs grpc changes

### PR DESCRIPTION
**Pull request by alenkani**
What does this Pull Request accomplish?
added API RFmxInstr_GetAvailablePaths

This adds the following:

Attributes
new API added :-
RFmxInstr_GetAvailablePaths

Why should this Pull Request be merged?
https://dev.azure.com/ni/DevCentral/_workitems/edit/2518973/

What testing has been done?
Locally built grpc-device.

### CODE REVIEW

File: source/codegen/metadata/nirfmxinstr/functions.py [hunk 1]
```diff
@@ -1314,6 +1314,36 @@
         ],
         'returns': 'int32'
     },
+    'GetAvailablePaths': {
```
Comment (astarche) [inline source/codegen/metadata/nirfmxinstr/functions.py:1317]: This has one of the changes from https://github.com/ni/grpc-device-scrapigen/pull/197 but not the others. Are you planning to update this one or make a separate pr?

Comment (alenkani) [reply source/codegen/metadata/nirfmxinstr/functions.py:1317]: https://github.com/ni/grpc-device/pull/990
https://github.com/ni/grpc-device/pull/981 
rest of the function are already added

**Review by reckenro — APPROVED** [review]
